src/api/app/models/workflow/step.rb

```diff
@@ -66,14 +68,19 @@ def source_project_name
     step_instructions[:source_project]
   end
 
-  def target_package_name
+  def target_package_name(short_commit_sha: false)
```

Comment:
Hm, why is this an option?

Reply:
Because there are places where we need the target package name but we don't want to get the short commit sha appended but the full one.